docs/sql_reference/src/asciidoc/_chapters/sql_statements.adoc

```diff
@@ -2963,18 +2963,22 @@ column-definition is:
 <<<
 ```
 data-type is:
-    char[acter] [(length [characters])]
-          [character set char-set-name]
-          [upshift] [[not]casespecific]
+    char[acter] [(length) [characters]]

```

Comment:
Should be "char[acter] [(length [characters])]"  (The trailing parenthesis should come after the optional [characters] syntactic element)

Reply:
Thanks for your eagle eye 😁 
Sorry it's my careless mistake :pig: I've corrected them all :pencil2: